skiko/src/awtMain/kotlin/org/jetbrains/skiko/SkiaLayer.awt.kt

```diff
@@ -291,7 +291,7 @@ actual open class SkiaLayer internal constructor(
                 redrawer = renderFactory.createRedrawer(this, renderApi, analytics, properties)
                 redrawer?.syncSize()
             } catch (e: RenderException) {
-                println(e.message)
+                Logger.warn("Fallback to next API: ${e.message}")
```

Comment:
This can happen often, actually. When we run an app on a VM, don't have drivers installed, and when we enable transparency on Windows (it supported only on OpenGL right now).

Let's change it to `info`

Reply:
> when we enable transparency on Windows (it supported only on OpenGL right now).

On the other hand we should switch to OpenGL straight away, without a fallback. In 2 other cases, it is probably should be `warn`.

Reply:
It's not a big difference for now, because we have not implemented log filtering yet. But still, for me this message is quite important and unexpected, so I prefer to leave it as `warn` for now.